python/lsst/ts/MTAirCompressor/aircompressor_csc.py

```diff
@@ -0,0 +1,262 @@
+# This file is part of ts_AirCompressors.
+#
+# Developed for the LSST Data Management System.
```

Comment:
Please change `LSST Data Management System` to `Vera Rubin Observatory Telescope and Site Systems` here and in all other files.